packages/react-tabster/src/hooks/useFocusIndicatorStyle.ts

```diff
@@ -30,18 +31,17 @@ const getFocusOutlineStyles = (options: FocusOutlineStyleOptions) => {
   const outlineOffsetRight = (outlineOffset as FocusOutlineOffset)?.right || outlineOffset;
 
   return {
-    borderColor: 'transparent',
+    ...shorthands.borderColor('transparent'),
     ':after': {
       content: '""',
       position: 'absolute',
       pointerEvents: 'none',
-      boxSizing: 'outline-box',
       zIndex: 1,
 
-      borderStyle: 'solid',
-      borderWidth: outlineWidth,
-      borderRadius: outlineRadius,
-      borderColor: outlineColor,
```

Comment:
We should use shorthand functions. CSS shorthands are not support by Griffel and will be stripped out from insertion with next update.